docassemble/AssemblyLine/al_document.py

```diff
@@ -594,6 +594,45 @@ def overflow_value(self, field_name:str, overflow_message:str=None, preserve_new
       overflow_message = self.default_overflow_message
     return self.overflow_fields[field_name].overflow_value(overflow_message=overflow_message, preserve_newlines=preserve_newlines, input_width=input_width)
 
+class ALStaticDocument(DAStaticFile, ALDocument):
+  """A class that allows one-line initialization of static documents to include in an ALDocumentBundle.
+  
+  Note:
+      You should always place the static file within the /data/static folder of a package.
+      ALDocumentBundle relies on a publically accessible file. The /data/templates folder is private.
+      
+  Attributes:
+      filename(str): the path to the file within /data/static/.
+      title(str): The title that will display as a row when invoked with `download_list_html()` method
+                  of an ALDocumentBundle.                  
+  Examples:
+      Add a static PDF file to a document bundle.
+      .. code-block:: yaml
+        ---
+        objects:
+          - static_test: ALStaticDocument.using(title="Static Test", filename="static.pdf", enabled=True)
+        ---
+        objects: 
+          - bundle: ALDocumentBundle.using(elements=[static_test], filename="bundle", title="Documents to download now")
+          
+  Todo:
+      Handle files placed in /data/templates if that turns out to be useful. Likely by copying into
+      a DAFile with pdf_concatenate().
+  """
+  def init(self, *pargs, **kwargs):
```

Comment:
I'm having issues getting the StaticFile to show up when you call `as_pdf` on a bundle. It's enabled, but it doesn't show up in the generated file, even if I use a PDF as the backing static file. Does `as_pdf` need to be overwritten too?

Reply:
I think the issue was the `__str__` method, but there were some other issues with PDFs. I'm just making it an extension of DAStaticFile for now, which resolves this issue.